content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -147,10 +74,13 @@ For example:
 
 ```shell
 # Label your nodes with the accelerator type they have.
-kubectl label nodes <node-with-k80> accelerator=nvidia-tesla-k80
-kubectl label nodes <node-with-p100> accelerator=nvidia-tesla-p100
+kubectl label nodes node1 accelerator=example-gpu-x100
+kubectl label nodes node2 accelerator=other-gpu-k915
```

Comment:
IMHO this part of documentation is also vendor specific, at least nowadays. NFD can be used to automatically label nodes based on what PCI devices they have.  Whereas manually labeling nodes by their purpose is not GPU specific.

Reply:
@eero-t what I'm trying to convey is that a cluster operator can label their nodes how they like so that Pod that require a specific kind of GPU get scheduled there. The alternative is that the scheduler guesses and perhaps eventually the Pod lands on the right kind of node by luck.

The values that a particular cluster operator might choose ought to be meaningful. If they happen to buy X100 GPUs from Example, Inc. then this labelling will work great. Same if they use k915 boards from Other AG.

The vendors are fictional but the mechanism isn't.

Reply:
My point was that manual labeling is basic k8s feature (not related to GPUs), so it does not need to be explained here.

In the Nvidia case you can use Nvidia version of NFD, which automatically adds GPU type labels to nodes:
https://github.com/NVIDIA/gpu-feature-discovery

And with other GPUs, suitable PCI ID rule can be added to standard NFD:
https://kubernetes-sigs.github.io/node-feature-discovery/stable/advanced/customization-guide.html#available-features

That way node labels would be always up to date without manual maintenance.

Reply:
> My point was that manual labeling is basic k8s feature (not related to GPUs), so it does not need to be explained here.

If we skip that part, the next step would be to delete the page as there is then nothing Kubernetes-specific left to retain.

Reply:
Please read the whole comment, not just first clause.  IMHO you should not be promoting manually labeling GPU types when k8s offers automatic device labeling add-on (node-feature-discovery), and vendors listed here provide their own GPU labeling services (AMD: node-labeller, Nvidia: gpu-feature-discovery).

Reply:
The purpose of this PR is: omit vendor install docs from scheduling task.

If we have a second purpose, to also revisit the docs about node labelling, then that is a different purpose. Each PR should aim to have a single purpose.

Reply:
> one should not be promoting manually labeling GPU types

Noted. We're not promoting one or other approach; instead, we're letting readers decide.

Reply:
Adding the missing options needed for that user decision, belonging to another PR -- fair enough.

AMD labeller instructions are vendor specific though, so I think removing most of that (except for telling what it is and pointing to its docs) would belong to this PR?